### ISSUE TITLE
CRIS and FAIR data infrastructure: a contribution by euroCRIS (Making FAIR work)

### ISSUE THREAD

**Issue by EdSimons** [FAIR-EG/consultation#26]
euroCRIS, representing  the research information community in general and the CRIS community in particular (http://www.eurocris.org)  is gladly willing to actively contribute to the EOSC initiative. The expertise and experience present within euroCRIS may bring added value for the realisation of an optimal EOSC, more specifically so for the FAIR aspect of the infrastructure. The rationale for this is summarized in the points below. 

1. The availability and interoperability of optimal metadata is crucial for the FAIR-ness of a research data infrastructure. This does not only regard metadata concerning the datasets as such, but also metadata, containing  information about objects and aspects related to the data(sets), such as: publications based upon the data, the project the data resulted from, researchers and institutes involved in the research, funders that provided the grants, controlled vocabularies classifying the research and its datasets, etc… Each of these related metadata provide extra entries that facilitate and promote the findability as well as - the interpretability of -  the (re)usefulness of the data and as such substantially enhance the FAIR-ness of the data. 

2. Typically  such a full and interrelated set of metadata about research and its products and objects is stored in CRIS: Current Research Information Systems. Therefore (the information in) CRIS are valuable resources for any research data infrastructure. 

3. As part of its activities, euroCRIS has developed a standard, 3-layer, architectural model describing and structuring the use of metadata in a research data infrastructure. In this model a distinction is made between “generic” metadata that apply to any type of research data(sets) and discipline- or subject-specific metadata. The model is summarized  in the attached figure:
[3LayerRD_Metadata_Model.pdf](https://github.com/FAIR-Data-EG/consultation/files/1172795/3LayerRD_Metadata_Model.pdf)

4. Two specific examples of CRIS and CERIF in handling research data sets are in progress – one in the UK with the ‘CERIFication of the Research Data Shared Service’ pilot project and the second in the Netherlands ("RDS Project") to describe, upload and archive research datasets by means of a CRIS (a project in cooperation with DANS, the national NL data hosting organisation) . 

In conclusion: the success of the European Open Science Cloud (EOSC) requires efficient and effective underpinning systems - among which CRIS - based on open standards and comprehensive interlinked metadata. The latter are essential to optimize the findability, interpretability and reusability of datasets as well as facilitate interoperability and so reduce the burden in collecting and reusing information. Together with authoritative persistent identifiers and standard definitions, a standard metadata exchange model provide the three pillars of interoperability. (see: http://dspacecris.eurocris.org/handle/11366/567)

**Comment by LeifLaaksonen** [comment]
Ed, thanks a lot for your constructive input. In the light of the great work you are doing we share very much the view that strong organisations like euroCRIS, with a solid technological background in RIs, should have a big impact on the successful implementation of FAIR. The euroCRIS linkage, support and involvement to the national stakeholders will be central and we know that euroCRIS can have a big impact in this process.
